src/tests/suite/daos_test_common.c

```diff
@@ -843,7 +843,7 @@ daos_dmg_pool_target(const char *sub_cmd, const uuid_t pool_uuid,
 	int		rc;
 
 	/* build and invoke dmg cmd */
-	dts_create_config(dmg_cmd, "dmg pool %s -i --pool=%s --rank=%d",
+	dts_create_config(dmg_cmd, "dmg pool %s -i --pool=" DF_UUIDF " --rank=%d",
```

Comment:
This appears to be creating a command line to use (although it's doing it in an odd way), so I used DF_UUIDF here which should print the whole string, so this shouldn't cause a regression.